variants/ViraLink-G0.1/pins_arduino.h

```diff
@@ -0,0 +1,42 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+static const uint8_t LED_BUILTIN = 5;
+#define BUILTIN_LED LED_BUILTIN
+#define LED_BUILTIN LED_BUILTIN
+
+static const uint8_t RESET_KEY = 0;
+
+static const uint8_t RS485_TX = 32;
+static const uint8_t RS485_RX = 35;
+
+static const uint8_t SDA = 4;
+static const uint8_t SCL = 16;
+
```

Comment:
Please provide default SPI pins. If you don't want to specify them, you can use -1 as shown in suggestion.
```suggestion
static const uint8_t SS = -1;
static const uint8_t MOSI = -1;
static const uint8_t SCK = -1;
static const uint8_t MISO = -1;
```